homeassistant/components/calendar/google.py

```diff
@@ -18,11 +18,11 @@
 
 DEFAULT_GOOGLE_SEARCH_PARAMS = {
     'orderBy': 'startTime',
-    'maxResults': 1,
+    'maxResults': 5,
     'singleEvents': True,
 }
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
```

Comment:
Were the 15 minutes previously chosen with some deeper reason in mind? Some API rate limit from Google?

Reply:
Oh, I forgot to set that back to 15. Thanks for the hint.